perf-event/examples/println-cpi-raw.rs

```diff
@@ -0,0 +1,46 @@
+use perf_event::events::{Raw, Software};
+use perf_event::{Builder, ReadFormat};
+
+fn main() -> std::io::Result<()> {
+    let insns_retired: Raw = Raw::new(0x08);
+    let cpu_cycles: Raw = Raw::new(0x11);
```

Comment:
Do you have a link or somewhere that describes where these values come from and what platform they're valid for? The example is useful but not likely to work on most people's machines. At the very minimum a comment describing what platform this is expected to work on would be useful.

Reply:
Ah. These are armv8 pmu events. I was using the Neoverse Technical reference: https://developer.arm.com/documentation/100616/0400/debug-descriptions/performance-monitor-unit/pmu-events.